src/parser/jobs/brd/CritRate.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * @author Yumiya
+ */
+
+import Module from 'parser/core/Module'
+import STATUSES from 'data/STATUSES'
+import ACTIONS from 'data/ACTIONS'
+
+// Relevant crit buffs
+const CRIT_MODIFIERS = [
+	{
+		id: STATUSES.BATTLE_LITANY.id,
+		strenght: 0.15,
+	},
+	{
+		id: STATUSES.CHAIN_STRATAGEM.id,
+		strenght: 0.15,
+	},
+	{
+		id: STATUSES.CRITICAL_UP.id,
+		strenght: 0.02,
+	},
+	{
+		id: STATUSES.THE_SPEAR.id,
+		//fuck royal road
```

Comment:
`core/Arcanum` tracks all RR effects and provides a strength modifier in the event.